components/Cards/src/MDCCardCollectionCell.h

```diff
@@ -37,6 +37,34 @@ typedef NS_ENUM(NSInteger, MDCCardCellState) {
   MDCCardCellStateSelected
 };
 
+/**
+ The horizontal alignment of the image when in selectable mode (`selectable` is set to YES).
+ */
+typedef NS_ENUM(NSInteger, MDCCardCellHorizontalImageAlignment) {
+  /** The alignment of the image is to the right of the card. */
+  MDCCardCellHorizontalImageAlignmentRight = 0,
+
+  /** The alignment of the image is to the center of the card. */
+  MDCCardCellHorizontalImageAlignmentCenter,
+
+  /** The alignment of the image is to the left of the card. */
+  MDCCardCellHorizontalImageAlignmentLeft,
```

Comment:
I agree with @romoore .  If we weren't going to expose alignment, I'd be happy to match Apple.  If we are going to expose alignment, we should include AlignmentLeading and AlignmentTrailing.
Ideally we will be able to migrate from this custom enumeration to UIControlContentHorizontalAlignment.  (When we hit 11+)

Reply:
If you want to open a TODO to get this landed, that seems completely reasonable, as it would break the RTL support into its own PR.